RZUtils/Categories/UIView/UIView+RZAutoLayoutHelpers.h

```diff
@@ -107,6 +107,62 @@
  */
 - (NSLayoutConstraint*)rz_pinnedCenterYConstraint;
 
+/**
+ *  Return the receiver's bottom-to-top constraints, if any exist.
+ *
+ *  @return The constraints or nil.
+ */
+- (NSArray *)rz_bottomToTopConstraints;
+
+/**
+ *  Return the receiver's top-to-bottom constraints, if any exist.
+ *
+ *  @return The constraints or nil.
+ */
+- (NSArray *)rz_topToBottomConstraints;
+
+/**
+ *  Return the receiver's left-to-right constraints, if any exist.
+ *
+ *  @return The constraints or nil.
+ */
+- (NSArray *)rz_leftToRightConstraints;
+
+/**
+ *  Return the receiver's right-to-left constraints, if any exist.
+ *
+ *  @return The constraints or nil.
+ */
+- (NSArray *)rz_rightToLeftConstraints;
+
+/**
+ *  Return the receiver's only bottom-to-top constraint.
+ *
+ *  @return The constraint or nil.
```

Comment:
Might want to note here that this throws an exception if there are multiple constraints.